api/Element.json

```diff
@@ -4194,7 +4226,7 @@
               {
                 "version_added": "12",
                 "version_removed": "16",
-                "notes": "This function is implemented in the <code>HTMLElement</code> API, meaning non-HTML elements (like SVG elements) cannot use this function."
+                "notes": "Only supported for <a href='https://developer.mozilla.org/docs/Web/API/HTMLElement'><code>HTMLElement</code></a>, not all <code>Element</code> objects, such as <a href='https://developer.mozilla.org/docs/Web/API/SVGElement'><code>SVGElement</code></a>."
```

Comment:
Good, more consistent notes. The old notes weren't wrong because this (`getElementsByClassName`) really is a function, but the new one works for both functions and properties.